cbmc/proofs/IotMqtt_SubscribeSync/cbmc-viewer.json

```diff
@@ -0,0 +1,18 @@
+{ "expected-missing-functions":
+  [
+    "IotClock_GetTimeMs",
+    "IotLog_Generic",
+    "IotMqtt_strerror",
+    "IotMutex_Destroy",
+    "IotMutex_Lock",
+    "IotMutex_Unlock",
+    "IotSemaphore_Create",
+    "IotSemaphore_Destroy",
+    "IotTaskPool_GetSystemTaskPool",
+    "_checkRetryLimit",
```

Comment:
Screaming out for a comment, which json won't let you do, but maybe add a blank line as separator just to draw attention to the distinction.

Reply:
Done! I also highlight them in the Makefile:
```Makefile
# We abstract these functions because manual inspection demonstrates they are unreachable
ABSTRACTIONS += --remove-function-body _checkRetryLimit
ABSTRACTIONS += --remove-function-body _scheduleCallback
ABSTRACTIONS += --remove-function-body _scheduleNextRetry
```